### PR TITLE
Restructure Getting Started docs

### PR DESCRIPTION
**Pull request by jbyers19**
### Proposed changes
This PR restructures our Getting Started docs to make them easier to follow.

image

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/nginx-service-mesh/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [x] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs/content/get-started/platform-setup/supported-platforms.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
 ---
 title: Supported Platforms
-description: Find out which Kubernetes platforms are supported for use with NGINX Service Mesh.
+description: Find out which platforms are supported for use with NGINX Service Mesh.
 date: 2020-08-17
 weight: 100
 toc: true
-docs: "DOCS-688"
```
Comment (sjberman) [inline docs/content/get-started/platform-setup/supported-platforms.md:7]: ?

File: docs/content/get-started/uninstall/uninstall.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+{{< /note >}}
+
+## Troubleshooting
+
+In some cases, the `remove` command may fail for unexpected reasons due to environmental, network, or timeout errors. If the `remove` command fails continually, manual intervention may be necessary.
+
+Run this command to see all resources associated with the mesh currently present in your cluster:
+
+```bash
+kubectl get pod,svc,sa,pvc,rolebinding,clusterrole,role,clusterrolebinding,crd,apiservices -l app.kubernetes.io/part-of==nginx-service-mesh -A
```
Comment (sjberman) [inline docs/content/get-started/uninstall/uninstall.md:64]: Same comment as before.

File: docs/content/get-started/platform-setup/supported-platforms.md (hunk 1 above)
Comment (jbyers19) [reply docs/content/get-started/platform-setup/supported-platforms.md:7]: 🤷

File: docs/netlify.toml [hunk 3]
```diff
@@ -42,3 +42,9 @@
   from = "*"
   to = "/nginx-service-mesh/404.html"
   status = 404
+
+[[redirects]]
+  from = "/nginx-service-mesh/guides/uninstall"
+  to = "/nginx-service-mesh/get-started/install/#uninstall"
```
[the remark this answers is not in the document]
Comment (jbyers19) [reply docs/netlify.toml]: Oh, didn't mean to commit that. With the new structure it's easy to find the uninstall and upgrade docs. Don't think we want to start down the path of adding a Netlify redirect every time we move a doc.

**Review by sjberman — APPROVED** [review]
LGTM pending review from docs team

**Review by ADubhlaoich — APPROVED** [review]
Looks good to me!

**Pull request merged**